_scripts/app1-6.R

```diff
@@ -0,0 +1,28 @@
+{
+  
+  rm(list=ls());
+  options(show.error.locations = TRUE);
+  
+ # Prompt user to input location 
+  
+  tempLoc <- readline("Where were the measurements recorded?: ");
+  
+ # Prompt user to input recorded measurements
+  
+  temp1 <- as.numeric(readline("Enter the first measurement: "));
+  temp2 <- as.numeric(readline("Enter the second measurement: "));
+  temp3 <- as.numeric(readline("Enter the third measurement: "));
+  temp4 <- as.numeric(readline("Enter the fourth measurement: "));
+  temp5 <- as.numeric(readline("Enter the fifth measurement: "));
+  
+  # Calculate mean temperature
+  
+  mnTemp <- sum(temp1, temp2, temp3, temp4, temp5)/5; # n = 5
+  
+  # Summarize in console
+  
+  cat("The temperature measurements are from: ", tempLoc,
```

Comment:
One thing about cat()
The commas are used to separate components put inside cat() and  a space as a separator in the output.  If you don't want a space, you can change this in the function by adding sep="" to the beginning:
cat(sep="", ...)  # no separator 
cat(sep=",", ...)  # a comma separator 
cat(sep="!@#$%", ...)  # !@#$% separator (probably not practical!)